packages/server/package.json

```diff
@@ -74,5 +74,9 @@
     "trumpet": "^1.7.2",
     "websocket": "^1.0.30",
     "yargs": "^15.0.2"
+  },
+  "volta": {
+    "node": "12.16.0",
+    "yarn": "1.19.1"
```

Comment:
The other packages have `volta` configurations, allowing you to work with the individual package as if it weren't in a monorepo. I've added this here to follow suit.

Related volta ticket: https://github.com/volta-cli/volta/issues/378